File: for loops/factorial.c
#include <stdio.h>

int main(void) {
	int n,i,factorial=1;
	printf("Enter the number n= \n");
	scanf("%d", &n);
	
	for(i=1;i<=n;i++){
		factorial=factorial*i;
		}
	printf("factorial of %d is = %d",n, factorial);
	return 0;
}

File: JAVA_ASSIGNMENT/factorial.java
import java.util.*;
public class code {
    public static void main(String[] args){
       int fact = 1;
       Scanner sc = new Scanner(System.in);
       System.out.print("Enter a number: ");
       int num = sc.nextInt(); 
       for(int i=1;i<=num;i++){
        fact = fact * i;
       }
       System.out.println("The factorial of "+ num + " is: "+fact);
    }
}
